<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>predict_pro</class>
 <widget class="QDialog" name="predict_pro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="9">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="32" column="11">
    <widget class="QComboBox" name="team2_hero5_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="20" column="11">
    <widget class="QComboBox" name="team2_hero4_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="27" column="6">
    <widget class="QLabel" name="winner_label">
     <property name="styleSheet">
      <string notr="true">font: 900 28pt &quot;Segoe UI Black&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="imgT1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>140</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="32" column="1">
    <widget class="QComboBox" name="team1_hero5_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="20" column="4">
    <widget class="QComboBox" name="team1_hero2_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="20" column="9">
    <widget class="QComboBox" name="team2_hero1_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="27" column="4">
    <widget class="QComboBox" name="team1_hero4_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="27" column="9">
    <widget class="QComboBox" name="team2_hero3_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="11">
    <widget class="QLabel" name="imgT2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>140</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="1">
    <widget class="QComboBox" name="team1_hero1_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="27" column="11">
    <widget class="QComboBox" name="team2_hero2_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="20" column="7">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="team1_select_menu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 2px solid #094065;
font: 14pt &quot;Sans Serif Collection&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Team 1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_pro">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 18pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>PRO MATCH</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>130</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="11">
    <widget class="QComboBox" name="team2_select_menu">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border: 2px solid #094065;
font: 14pt &quot;Sans Serif Collection&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>Team 2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="34" column="6">
    <widget class="QLabel" name="predict_team">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="31" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="10">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="27" column="1">
    <widget class="QComboBox" name="team1_hero3_select_menu">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 5px;       
border: 2px solid #094065;
font: 8pt &quot;Sans Serif Collection&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="20" column="6">
    <widget class="QPushButton" name="predict_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 20px;            
border: 2px solid #094065;
font: 14pt &quot;Sans Serif Collection&quot;;</string>
     </property>
     <property name="text">
      <string>Predict</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
